wordpress-mobile/WordPress-iOS
# Replace Gravatar with GravatarURL

Part of effort: https://github.com/wordpress-mobile/WordPress-iOS/issues/22543

Replaces the deprecated Gravatar struct with GravatarURL in:

CommentContentTableViewCell
NoteBlockCommentTableViewCell
NoteBlockHeaderTableViewCell
NoteBlockUserTableViewCell
PeopleCell
PersonViewController
RevisionsTableViewCell
NotificationContentView

To test:

Prerequsities:

Make sure you check out the branch `task/gravatar-integration` in your local WordPressUI, and the latest `trunk` in your local Gravatar repo.

Testing:

This is a simple change so smoke test a few places we have touched and see that avatars are loading. I documented the testing steps [in this comment](https://github.com/wordpress-mobile/WordPress-iOS/issues/22543#issuecomment-1948014118) for future usage. You can search there the items in the list above ☝️.

NotificationContentView is exceptional, we need to receive push notifications for that. So we can skip it for now and test at the end of the whole effort maybe.

We are going to test these places more throughly in the consequential PRs that replace the download operation with SDK methods.

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
